object/search_test.go

```diff
@@ -187,7 +187,7 @@ func TestSearchFilters_AddObjectIDFilter(t *testing.T) {
 }
 
 func TestSearchFilters_AddSplitIDFilter(t *testing.T) {
-	id := object.NewSplitID()
+	id := *object.NewSplitID()
```

Comment:
do not return a pointer then?

Reply:
that's a good point, actually i wanted to do this separately: there are some other places returning pointer w/o any sense. Will add commit with global change

Reply:
this PR? another issue?

Reply:
done